testdata/terraform/cloudflare_waf_override/test.tf

```diff
@@ -1,7 +1,7 @@
 resource "cloudflare_waf_override" "terraform_managed_resource" {
   description = "Enable Cloudflare Magento ruleset for shop.example.com"
   groups = {
-    "ea8687e59929c1fd05ba97574ad43f77" = "default"
+    ea8687e59929c1fd05ba97574ad43f77 = "default"
```

Comment:
if this starts with a number, it needs to be quoted. hopefully hclwrite handles that.

Reply:
i'm hopeful it will. there is another attribute in here that follows that, and it quotes it. so 🤞🏼

Reply:
https://github.com/cloudflare/cf-terraforming/blob/master/testdata/terraform/cloudflare_waf_override/test.tf#L14